content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Applying Pod Security Standards at Cluster level
+content_type: tutorial
+weight: 10
+---
+
+{{% alert title="Note" %}}
+This tutorial applies only for new clusters.
+{{% /alert %}}
+
+Since v1.23, Pod Security admission (PSA) is enabled by default as it [graduated
+to beta](blog/_posts/2021-12-15-pod-security-admission-beta.md). In this 
+tutorial, we will enforce `baseline` Pod Security Standard, at a cluster level.
+
+## {{% heading "prerequisites" %}}
+
+Install the following on your workstation:
+
+- [KinD](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+# Finding the right Pod Security Standard
+
+As of
+v1.23, [Pod Security Admission](/docs/concepts/security/pod-security-admission/)
+allows usage of
+built-in [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
+that can be set in three different modes: `enforce`, `audit`, `warn`.
+
+Following steps allows gathering information about which Pod Security Standards
+will be most appropriate to apply:
+
+1. Create a cluster without any pod security standard applied:
+
+    ```
+    $ kind create cluster --name psa-wo-cluster-pss --image kindest/node:latest
+    Creating cluster "psa-wo-cluster-pss" ...
+    ✓ Ensuring node image (kindest/node:latest) 🖼
+    ✓ Preparing nodes 📦  
+    ✓ Writing configuration 📜
+    ✓ Starting control-plane 🕹️
+    ✓ Installing CNI 🔌
+    ✓ Installing StorageClass 💾
+    Set kubectl context to "kind-psa-wo-cluster-pss"
+    You can now use your cluster with:
+    
+    kubectl cluster-info --context kind-psa-wo-cluster-pss
+    
+    Thanks for using kind! 😊
+    
+    ```
+
+2. Switch kubectl context to point it to this new cluster
+    ```
+    $ kubectl cluster-info --context kind-psa-wo-cluster-pss
+    Kubernetes control plane is running at https://127.0.0.1:61350
+    CoreDNS is running at https://127.0.0.1:61350/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+    
+    To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+    ```
+
+3. Let's look at existing namespaces in the cluster:
+
+    ```
```

Comment:
Separate output from command (follow the same style as previous comments)

Reply:
done